### PR TITLE
Target Specific CML version on created instance.

### PR DESCRIPTION
**Pull request by dacbd**
The default should be the same as the invoking CML version or unset.
Option `hidden` as this value probably shouldn't be set 98%

### CODE REVIEW

**Comment by dacbd** [comment]
An Example of the convenience factor this could provide; testing #689 against #808 in a live setting if I could get it to happen twice 🔍

File: bin/cml/runner.js [hunk 1]
```diff
@@ -553,6 +556,12 @@ exports.builder = (yargs) =>
         description: 'Specifies the subnet to use within AWS',
         alias: 'cloud-aws-subnet-id'
       },
+      cmlVersion: {
+        type: 'string',
+        default: require('../../package.json').version,
```
Comment (casperdcl) [inline bin/cml/runner.js:561]: :eyes: is it safe to assume this works? I doubt it, esp. for the binary distributions

Comment (0x2b3bfa0) [reply bin/cml/runner.js:561]: Yes, even for the binary distributions. Vercel [`pkg`](https://github.com/vercel/pkg) looks for calls to the `require` function and adapts them appropriately to the virtual file system.

Comment (0x2b3bfa0) [reply bin/cml/runner.js:561]: (Tested and confirmed)

File: bin/cml/runner.js [hunk 2]
```diff
@@ -553,6 +556,12 @@ exports.builder = (yargs) =>
...
+        description: 'CML version to load on tpi instance',
```
Comment (0x2b3bfa0) [inline bin/cml/runner.js:562]:
```suggestion
        description: 'CML version to load on TPI instance',
```

**Comment by 0x2b3bfa0** [comment]
Looks great! @dacbd, can you please include this option in `terraform.test.js` snapshot tests and update them?

**Comment by dacbd** [comment]
Will do.

**Comment by dacbd** [comment]
This test failure looks to be unrelated to any of the changes? https://github.com/iterative/cml/runs/5428037240?check_suite_focus=true#step:7:81

**Comment by 0x2b3bfa0** [comment]
Yes, it looks like our tests are broken (?) 🔔 @iterative/cml

**Comment by casperdcl** [comment]
Looks like [GL token length change from 20 to 29](https://github.com/iterative/cml/runs/5440254790?check_suite_focus=true#step:7:76)?

**Comment by DavidGOrtega** [comment]
Yes. Its something that I also saw in #689

**Comment by DavidGOrtega** [comment]
@dacbd https://github.com/iterative/cml/blob/266e6126e66b8e5119290b4c3c083cbf71c47c93/src/drivers/gitlab.test.js#L43

**Comment by dacbd** [comment]
> @dacbd
> 
> https://github.com/iterative/cml/blob/266e6126e66b8e5119290b4c3c083cbf71c47c93/src/drivers/gitlab.test.js#L43

thanks, cherry-picked over.

**Review by casperdcl — APPROVED** [review]

**Comment by DavidGOrtega** [comment]
Thanks @dacbd !!! ❤️

**Comment by 0x2b3bfa0** [comment]
Ditto! ❤️

**Pull request merged**